docs/design/HealthEndpoint.md

```diff
@@ -53,54 +63,53 @@ We need to update the dab config file to include details of the health check for
 
 #### `runtime.health` Configuration
 
-The runtime configuration would include details like cache-ttl in case we need to cache the response of health checks, the max-drop value which tells you the degree of parallelism i.e. how many queries that DAB shoud run at once to get health results and roles i.e. which role is allowed to view the health information of DAB.
+The runtime configuration would include details like cache-ttl in case we need to cache the response of health checks, the max-dop value which specifies the degree of parallelism i.e. how many queries that DAB should run at once to get health results and roles i.e. which role is allowed to view the health information of DAB.
 
 | **Property**   | **Data Type** | **Required** | **Default** | **Description**                                                                                      |
 |----------------|---------------|--------------|-------------|------------------------------------------------------------------------------------------------------|
-| `enabled`     | Boolean       | No           | `true`      | Enables or disables health checks at the runtime level.                                             |
-| `cache-ttl`   | Integer       | No           | `5`         | Time-to-live (in seconds) for caching health check results.                                          |
+| `enabled`     | Boolean       | No           | `false`      | Enables or disables the comprehensive health checks for DAB Engine.                                             |
+| `cache-ttl`   | Integer       | No           | `5`         | Time-to-live (in seconds) for caching health check results. If this value is not specified, caching would not be enabled.                                          |
 | `max-dop`     | Integer       | No           | `1`         | Maximum Degree of Parallelism for running health checks.                                             |
-| `roles`       | Array         | No           | `*`         | Roles allowed to access the health endpoint (e.g., `anonymous`, `authenticated`).                   |
+| `roles`       | Array of strings         | Yes           | NA         | Roles allowed to access the health endpoint (e.g., `anonymous`, `authenticated`).                   |
 
 #### `data-source.health` Configuration
 
-The data source config parameters specify the query that we should run on the data source and the threshold of ms if should come under to qualify as a healthy data source for DAB.
+The data source config parameters specify the query that we should run on the data source and the threshold of ms it should come under to qualify as a healthy data source for DAB.
```

Comment:
If we removed the query, why does this section still say that the `query` will be specified?